### PR TITLE
Add System.IdentityModel.Tokens.Jwt and dependencies

### PR DESCRIPTION
**Pull request by captainsafia**
Add System.IdentityModel.Tokens.Jwt and dependencies to resolve https://github.com/dotnet/aspnetcore/issues/42599

### CODE REVIEW

**Comment by crummel** [comment]
@captainsafia I fixed up the packages output directory so the new package is included in the intermediate package.  From our side I think the last remaining issue is the version number of the package (currently `0.0.1`), what version should we be producing?

File: repos/azure-activedirectory-identitymodel-extensions-for-dotnet.proj [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          EnvironmentVariables="@(EnvironmentVariables)"
+          WorkingDirectory="$(ProjectDirectory)"
+          IgnoreStandardErrorWarningFormat="true" />
+
+    <Exec Command="$(DotnetToolCommand) build /bl:build.binlog $(SystemIdentityModelTokensJwtProject) $(BuildCommandArgs)"
+          EnvironmentVariables="@(EnvironmentVariables)"
+          WorkingDirectory="$(ProjectDirectory)"
+          IgnoreStandardErrorWarningFormat="true" />
+
+    <Exec Command="$(DotnetToolCommand) pack /bl:pack.binlog $(SystemIdentityModelTokensJwtProject) $(PackCommandArgs)"
```
Comment (crummel) [inline repos/azure-activedirectory-identitymodel-extensions-for-dotnet.proj:37]:
```suggestion
    <Exec Command="$(DotnetToolCommand) pack /bl:pack_TokensJwt.binlog $(SystemIdentityModelTokensJwtProject) $(PackCommandArgs)"
```

These three commands with `pack.binlog` will overwrite each other I believe.  We can break them out to preserve the logs for each of the pack actions.

File: repos/azure-activedirectory-identitymodel-extensions-for-dotnet.proj [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          EnvironmentVariables="@(EnvironmentVariables)"
+          WorkingDirectory="$(ProjectDirectory)"
+          IgnoreStandardErrorWarningFormat="true" />
+
+    <Exec Command="$(DotnetToolCommand) pack /bl:pack.binlog $(MicrosoftIdentityModelsTokenProject) $(PackCommandArgs)"
```
Comment (crummel) [inline repos/azure-activedirectory-identitymodel-extensions-for-dotnet.proj:42]:
```suggestion
    <Exec Command="$(DotnetToolCommand) pack /bl:pack_Tokens.binlog $(MicrosoftIdentityModelsTokenProject) $(PackCommandArgs)"
```

File: repos/azure-activedirectory-identitymodel-extensions-for-dotnet.proj [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          EnvironmentVariables="@(EnvironmentVariables)"
+          WorkingDirectory="$(ProjectDirectory)"
+          IgnoreStandardErrorWarningFormat="true" />
+
+    <Exec Command="$(DotnetToolCommand) pack /bl:pack.binlog $(MicrosoftIdentityModelJsonWebTokensProject) $(PackCommandArgs)"
```
Comment (crummel) [inline repos/azure-activedirectory-identitymodel-extensions-for-dotnet.proj:47]:
```suggestion
    <Exec Command="$(DotnetToolCommand) pack /bl:pack_JsonWebTokens.binlog $(MicrosoftIdentityModelJsonWebTokensProject) $(PackCommandArgs)"
```

**Review by crummel — COMMENTED** [review]
One small issue with the log files, otherwise LGTM.  Thanks!

File: repos/azure-activedirectory-identitymodel-extensions-for-dotnet.proj (hunk 1 above)
Comment (captainsafia) [reply repos/azure-activedirectory-identitymodel-extensions-for-dotnet.proj:37]: Good call!

**Review by crummel — APPROVED** [review]

**Review by MichaelSimons — APPROVED** [review]

**Pull request merged**